release-notes.md

```diff
@@ -16,6 +17,19 @@ The following updates were made in this release:
 
 * Improved existing log messages and added additional logging to assist with debugging.
 
+## January 28, 2019
+
+The following updates were made in this release:
+
+**iOS Griffon 1.0.4** 
+
+* Griffon SDK adds and reports uniqueIdentifier and timestamp associated with eventHub events. (works from Core v2.4.0).
```

Comment:
Please reword as follows:
"* The Griffon SDK now adds and reports the `uniqueIdentifier` and `timestamp` that are associated with eventHub events. 
     This update is effective starting with Mobile Core version 2.4.0.